share/instruments/instruments.xml

```diff
@@ -1939,25 +2328,34 @@
                   </Channel>
                   <genre>concertband</genre>
             </Instrument>
-            <Instrument id="c-clarinet">
+            <Instrument id="eb-clarinet">
                   <family>clarinets</family>
-                  <longName>C Clarinet</longName>
-                  <shortName>C Cl.</shortName>
-                  <description>C Clarinet</description>
-                  <musicXMLid>wind.reed.clarinet</musicXMLid>
+                  <trackName>Clarinet</trackName>
+                  <longName>Clarinet</longName>
+                  <shortName>Cl.</shortName>
+                  <dropdownName meaning="transposition">E♭</dropdownName>
```

Comment:
What you say is correct according to (one) historic classification:

Sopranino clarinets: Eb, D
Soprano clarinets: Bb, A, C

But these days it is more common to classify all these as soprano clarinets (but call them just "clarinets") and reserve the term "sopranino clarinet" for even higher instruments.

In any case, the distinction is arbitrary. [Here](https://en.wikipedia.org/wiki/A-flat_clarinet#/media/File:Aflat_eflat_bflat.jpg) is a picture of an Eb Clarinet (middle) and a Bb clarinet (right). While the Eb and Bb are different sizes, they are otherwise identical (same shape and same arrangement of holes and buttons/keys), so they are indeed just transposed versions of each other. I expect that a person who can already play one of them would get used to the size difference sooner than they would get used to reading notes in a different transposition.

[Here](https://en.wikipedia.org/wiki/Alto_clarinet#/media/File:Yamaha_Alto_Clarinet_YCL-631_II.jpg) is a picture of an Alto Clarinet. The Alto Clarinet has a slightly different shape to the others, but is still similar enough that we might *consider* grouping it with the other clarinets if there was an elegant way to do so. We want to avoid the system used in Dorico that introduces an additional column in the New Score Wizard:

Woodwind > Clarinets > Alto Clarinet

Therefore we have chosen to group them by name and just have a small dropdown to choose transposition.

Reply:
Oh, I'm playing the clarinet for more than 50 years now, mainly Bb/A but also used to play the Eb in a concert band for 5, 6 years. A brother plays the bass clarinet, I'm familiar with different kind of clarinets and their usages :smiley:

My point is, although the Eb flat looks similar as the Bb clarinet, unlike the alto clarinet, all 3 (Eb, Bb and alto) are **different** instruments in both parts and players. It is fine to group the A and the Bb clarinet but include the Eb clarinet in that group, just because of the historical naming, seems a bad idea to me. I can't think of a situation where somebody decides to add a clarinet and then thinks, what shall I use, an Eb, or a Bb clarinet. This choice is made as conscientious  as deciding whether to use an alto of a tenor saxophone. 
As a musician, I can't explain why A, Bb, Eb, ... are grouped and not the saxophones.

When grouping instruments, I think the musical background and usage of the instrument should be leading, not the actual name which is often more historical then "correct". So, the D and Eb clarinet can be one group, C, Bb and A can be another group.